src/test/java/seedu/address/logic/commands/ViewTagCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.ViewTagCommand.MESSAGE_VIEW_TAG_NONE;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ViewTagCommandTest {
+    @Test
+    public void execute_viewTagNone_success() {
+        Model model = new ModelManager();
+        Model expectedModel = new ModelManager();
+        CommandResult expectedCommandResult = new CommandResult(MESSAGE_VIEW_TAG_NONE, false, false);
```

Comment:
For the test cases, following the convention of the previously written cases, i think have to write out the string manually using String.format like [EditCommandTest](https://github.com/AY2324S1-CS2103T-W11-1/tp/blob/master/src/test/java/seedu/address/logic/commands/EditCommandTest.java#L44) line 44.

Reply:
I think we can do this for the viewTagNone test since the output is a constant. For the viewTag_success test, the output is a concatenation of tags and line breaks so it might be hard to hardcode the String output.

Please let me know if there's an easier way to do this and thank you for the comments.